### ISSUE TITLE
CIA Builder/Decryptor Fails with Sun/Moon

### ISSUE THREAD

**Issue by Kurausukun** [d0k3/Decrypt9WIP#117]
I made a thread over on GBATemp about this, but I wasn't able to resolve it over there, so I'm reporting it here just in case it is a legitimate issue. However, I am still not convinced the problem is not on my end.

I have a Japanese New 3DS with Luma CFW and A9LH. I preloaded Pokemon Moon from the eShop a couple days before release day, and now I'm trying to dump and decrypt a .cia of it. When I use the .cia builder function in Decrypt9WIP, it always builds the .cia successfully, but it can never properly decrypt it. I've tried dumping it with the new auto-decrypt feature that was added to the program recently, but that fails. I also tried dumping it without decrypting and then using the cia decryptor (deep) functionality separately, but that also fails.

Basically, like I said, I'm not sure what the issue is, or if it's a problem with how I'm dumping or if D9WIP is doing something wrong. Is there anything that I should be checking? I have a complete set of keys and an updated seeddb, so I don't think that's the issue.

**Comment by urherenow** [comment]
Updated seeddb from where? Are you really sure you have avalid seed for it?

**Comment by urherenow** [comment]
Or do you have dev unit stuff turned on in Luma's settings?

**Comment by d0k3** [comment]
Secoding that, you may most likely have a wrong seed for that game. Until the seed is released by big N (during the preload phase it is not released), you can't play nor decrypt the game.

**Comment by Committee** [comment]
Just installed SuMo from the freeshop/Ciangel and was able to decrypt and create CIA.

**Comment by audiohacked** [comment]
I've used Decrypt9WIP to decrypt and create CIA of Moon and Sun with no problems.

**Comment by Kurausukun** [comment]
All I did was use Decrypt9WIP's update seeddb function. I don't really know how seeds work very well. I figured I'd have one since I installed the game from the eShop, but maybe I'm thinking of tickets? How would I get the seed if that's what I'm missing?

**Comment by urherenow** [comment]
Just visit the shop again. You most likely went there BEFORE the official release. If you visit again afterwords, you'll get the seed, provided it thinks you have a valid ticket.

**Comment by Kurausukun** [comment]
Yes, I am talking about the whole game, and yes, I didn't visit the eShop again afterwards. I just log in again, and it'll give me the seed, then I should update my seeddb, right? I'll give it a try, thanks.

**Comment by ASleepyCat** [comment]
I also have this problem. It fails during the ExeFS and RomFS verification process, but ExeHdr passes it.

**Comment by d0k3** [comment]
Okay, seems pretty clear you guys had a wrong seed for that game. Feel free to reopen if there is another issue, but for now I'm closing this.

**Comment by Kurausukun** [comment]
Sorry, but I need to keep this alive--going to the eShop didn't work. And since it's been asked multiple times: no, I don't have any dev options enabled in Luma.

I deleted my old seeddb.bin, opened the eShop, built seeddb.bin again from Decrypt9WIP, and tried building again, but the exact same error happened. I see that ThunderRemix opened his own issue about this, and my output looks pretty much exactly the same, except there's a line about "console-unique info found, wiping" or something like that. Does preloading the game give you a dummy seed that doesn't work or something? If I deleted the game and reinstalled from the eShop, would it work then?
